lib/components/general_app_bar.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/cupertino.dart';
+
+import 'package:flutter/material.dart';
+import 'package:my_app/components/icon_label_button.dart';
+
+class GeneralAppBar extends StatelessWidget implements PreferredSizeWidget {
+  final Widget title;
+  final List<Widget> leftActions;
+  final List<Widget> rightActions;
+
+  const GeneralAppBar({
+    required this.title,
+    this.leftActions = const [],
+    this.rightActions = const [],
+  });
+
+  openSortOptionsPane() {}
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 44,
+      decoration: const BoxDecoration(
+        border: Border(
+          bottom: BorderSide(
+            width: 1,
+            color: Color.fromRGBO(204, 221, 255, 0.1),
+          ),
+        ),
+      ),
+      child: Row(children: [
+        Expanded(
+          child: Padding(
+            padding: const EdgeInsets.only(left: 11),
+            child: Row(children: leftActions),
+          ),
+        ),
+        title,
```

Comment:
I think it will be centered because I am using Expanded for the left and right area. Since both of them have the same flex value, the remaining space of the Row(after space occupied by the Text) will be equally distributed for the left and right action area.
![image](https://user-images.githubusercontent.com/19612629/191592723-7e57bec9-6984-4c9f-b3a5-36054d5eeabd.png)

Reply:
makes sense, cool